packages/rum-core/test/common/url.spec.js

```diff
@@ -269,3 +269,73 @@ describe('Url parser', function() {
     )
   })
 })
+
+/**
+ * In IE 11 - window.URL is not supported but it exists as an object
+ * Run these tests only on supported platforms
+ */
+if (window.URL.toString().indexOf('native code') !== -1) {
```

Comment:
We might have to skip this run on IE and Edge. Probably we can run only on chrome and ignore others.

Reply:
Ignored the test right now on IE and Edge. Test run on rest of the browsers.